README.md

```diff
@@ -138,7 +138,6 @@ If you have a base path of `/api` and your path is defined as `hello`, then a re
 
 ## TODO
 
-- [ ] OpenAPI v3.0 support
```

Comment:
Seems a little bold to consider this completely done before we add the tests?

Reply:
Yeah but I thought that's not really a ToDo — it's in Progress, hopefully close to completion. We're specifying that up in the Readme, over the first lines.